src/styles/card.css

```diff
@@ -0,0 +1,115 @@
+@import "./stars.css";
+
+.textSmall {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+}
+
+.textMedium {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.textLarge {
+  font-size: 16px;
+  font-weight: 700;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card {
+  flex-direction: column;
+
+  display: flex;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
+}
```

Comment:
You may consider writing width them margin and them display properties. It's a suggestion.